pkg/kernel/patches-preempt-rt-5.10.x/0001-sched-core-make-migrate_disable-and-migrate_enable-a.patch

```diff
@@ -0,0 +1,38 @@
+From b1a43659a2601f5dd1575084b617f8fdb52c030c Mon Sep 17 00:00:00 2001
+From: Roman Penyaev <r.peniaev@gmail.com>
+Date: Tue, 23 May 2023 13:40:31 +0200
+Subject: [PATCH 1/1] sched/core: make migrate_disable() and migrate_enable()
+ as non-GPL
+
+OpenZFS uses those and OpenZFS is not GPL-like.
+
+Signed-off-by: Roman Penyaev <r.peniaev@gmail.com>
+---
+ kernel/sched/core.c | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/kernel/sched/core.c b/kernel/sched/core.c
+index 1bc5a58bc18d..b9d2b87ffe99 100644
+--- a/kernel/sched/core.c
++++ b/kernel/sched/core.c
+@@ -1813,7 +1813,7 @@ void migrate_disable(void)
+ 	preempt_lazy_disable();
+ 	preempt_enable();
+ }
+-EXPORT_SYMBOL_GPL(migrate_disable);
++EXPORT_SYMBOL(migrate_disable);
```

Comment:
Are we sure that from a law/license point of view this is allowed?

Reply:
Until we don't ship the product - why not?

Reply:
Or even we ship the product

Reply:
Might be worth to ask a lawyer ...

Reply:
If this raises questions I can drop the compilation of the openzfs for preempt_rt.

Reply:
@eriknordmark interesting question from Christoph